### PR TITLE
Migrating AzDO governance scripts to banking-app

### LINKED ISSUE #65: Generalize existing governance scripts for ccf-app-samples
We have existing scripts for the banking-app

1. Create member + activate
2. Create user and activate
3. Submit proposal to deploy app to mCCF

Acceptance Criteria:
1. Move AzDO scripts to ccf-app-samples
2. Generalize & configure scripts - so they can be used for banking-app and data-reconciliation-app (Config file per sample app?) 
3. Test generalized governance scripts on banking-app in mCCF

### PR DESCRIPTION
**Pull request by julioalex-rezende**
Fixes #65

A lot has changed since opening this PR. please review in full

Including **governance** scripts to data-reconciliation app. They have been added to their corresponding folders based on what they are used for.
Az Keyvault scripts migration is pending, given that it would add changes that would go off-scope. We can handle it in a different task when we have this aligned

# Main changes
- Folder structure was changed (when needed) to reflect decisions made in data-reconciliation-app.
- `add_member.sh`, `add_user.sh` and `submit_proposal.sh` are generic. Therefore they are added to the root folder (root/scripts
- `test_sandbox.sh` and `test_docker.sh` are now generic to any application. Therefore they were moved to the root folder (root/scripts)
- `setup_governance.sh` is app-specific, so it was kept inside the application folder  (app/governance/scripts)
- `test.sh` script handles app-specific tests, so it was kept inside the application's folder (app/test)

# Next Steps
Incorporate scripts into data-reconciliation-app test

### CODE REVIEW

File: data-reconciliation-app/governance/scripts/add_member.sh [hunk 1]
```diff
@@ -0,0 +1,97 @@
+#!/bin/bash
```
Comment (ross-p-smith) [inline data-reconciliation-app/governance/scripts/add_member.sh:1]:
```suggestion
#!/bin/bash
set -euo pipefail
```

File: data-reconciliation-app/governance/scripts/add_user.sh [hunk 2]
```diff
@@ -0,0 +1,84 @@
+#!/bin/bash
```
Comment (ross-p-smith) [inline data-reconciliation-app/governance/scripts/add_user.sh:1]:
```suggestion
#!/bin/bash
set -euo pipefail
```

**Comment by ross-p-smith** [comment]
Is the intention just getting the scripts into the repo - or will you rewrite some of the existing scripts to make use of these functions? I'm a fan of the later. As far as your question - these scripts look generic across all samples, so I have a preference for not having them in each sample, but I have not mentally thought about how they will be "source"'d into each script. Maybe the make file will help?

**Comment by julioalex-rezende** [comment]
> Is the intention just getting the scripts into the repo - or will you rewrite some of the existing scripts to make use of these functions?

Thanks for the input @ross-p-smith . I'll keep working on the same PR and add the functionality that uses the scripts. My initial thoughts were on putting them in the repo and using them later, to make the next PR targeting the functionality. But I agree with your vision on this.

**Review by ross-p-smith — APPROVED** [review]
❤️

**Review by takuro-sato — APPROVED** [review]

**Pull request merged**